[BUG] Failed to start or build when using database provider other than mongodb

## Describe the bug

Run `npm run start`, `npm run start:dev`, or `npm run build` always failed when using mysql or postgresql database provider

## **To Reproduce**
Steps to reproduce the behavior:
1. In `.env` file, set `DB_PROVIDER` to 'mysql' or 'postgresql' and update the `DB_URL`.
2. Run `npm run db:migrate`.
3. Run `npm run db:seeder`.
4. Run `npm run start`, `npm run start:dev`, or `npm run build`.
5. See the error.

## Expected behavior

Successful build and start the app.

## Screenshots or Error logs

```sh
prisma/mongodb/seeder.ts:14:30 - error TS2339: Property '$runCommandRaw' does not exist on type 'PrismaClient<PrismaClientOptions, never, RejectOnNotFound | RejectPerOperation>'.

14     return await this.prisma.$runCommandRaw({
                                ~~~~~~~~~~~~~~

Found 1 error(s).
```

## Additional context

Ignore the TypeScript checking in `prisma/mongodb/seeder.ts:14:30` can solve this issue, since `$runCommandRaw()` method only available for mongodb provider. See the details about `$runCommandRaw()` method [here](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#runcommandraw).

--- a/prisma/mongodb/seeder.ts
+++ b/prisma/mongodb/seeder.ts
@@ -11,6 +11,9 @@ export class MongodbSeeder extends Seeder {
    * Delete all data in a collection.
    */
   protected async deleteCollection(collection: Areas) {
+    // Skip TypeScript checking because `$runCommandRaw()` method only available for mongodb provider.
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
     return await this.prisma.$runCommandRaw({
       delete: collection,
       deletes: [

